src/ui_fsmenu/options.h

```diff
@@ -121,11 +121,27 @@ class Options : public UI::Window {
 	void add_languages_to_list(const std::string& current_locale);
 	void update_language_stats();
 
+	void add_screen_resolutions(const OptionsCtrl::OptionsStruct& opt);
+
 	// Saves the options and reloads the active tab
 	void clicked_apply();
 	// Restores old options when canceled
 	void clicked_cancel();
 
+	// Data model for the screen resolution dropdown
+	class ScreenResolution {
+	public:
+		int32_t xres;
+		int32_t yres;
+		int32_t depth;
+		inline bool operator==(const ScreenResolution& x) const {
+			return xres == x.xres && yres == x.yres;
```

Comment:
Should this also compare `depth`?

Reply:
Actually I am more in favor of deleting depth as it is only written, never read. Maybe it was used once upon a time but is not anymore? What is your opinion?

Reply:
Okay let's delete it. Shouldn't matter much on modern hardware…